routes/schemas.js

```diff
@@ -119,7 +119,18 @@ module.exports = {
 			eventSubmission: Joi.object({ allow: Joi.array().items(Joi.object()) }),
 			nonce: Joi.string(),
 			created: Joi.number(),
-			activeFrom: Joi.number()
+			activeFrom: Joi.number(),
+			priceMultiplicationRules: Joi.array().items(
+				Joi.object({
+					multiplier: Joi.number().precision(10), // max 10 decimal places
+					amount: numericString,
+					evType: Joi.array().items(Joi.string().lowercase()),
+					country: Joi.array().items(Joi.string().lowercase()),
+					publisher: Joi.array().items(Joi.string().lowercase()),
```

Comment:
I suggest publishers are checksummed to be consistent with everything else now..so don't lowercase it